### DISCUSSION TITLE
我是流萤的狗！！！

### DISCUSSION THREAD

**Discussion by ususwcsm** [share121/inter-knot#5455] [崩坏：星穹铁道]
![img_1723332572_1711557802313](https://github.com/user-attachments/assets/cffad4a6-5dcd-4ac0-bf64-928d2f2f3c90)
![img_1405086032_1711557878430](https://github.com/user-attachments/assets/8506763c-654e-4020-bb81-c551f7df9474)

**Comment by fzdqz** [comment]
和流萤逛街的时候牵你<img src="https://bbs-static.miyoushe.com/static/2024/06/19/cd8d307076ea29a6095f34e40dc5b5f8_4644832424539789103.png" alt="流萤 比心" width="75" >

**Comment by shirekemi** [comment]
那你一定知道流萤是白色的吧
